bugRzilla_review.Rmd

```diff
@@ -168,45 +215,659 @@ Also there are columns which are empty so they are of no use of the analysis lik
+<font>
+
+```{r cre_last_week_graph}
+cre_last_week_graph <- cre_last_week %>%
+    ggplot() +
+    geom_line(aes(x = creation_week, 
+                  y = Bug_count, 
+                  color="creation_week")) +
+    geom_line(aes(x = lastdiffed_week, 
+                  y = Bug_count.1,
+                  color="lastdiffed_week")) +
+    labs(
+        title = "Week in which bugs are Created vs Last modified",
+        x = "Week",
+        y = "Bug_Count"
+    )
+
+ggplotly(cre_last_week_graph)
+```
+<font size = 4>
+    The Time-series graph is the relationship between the `bug_count` and the `week` in which the bug is `created` and `last_modified`. The aim is to check in every week how many bugs are `created` and `last modified`. In the first week of `May 2016`, a total of `32` bugs are created. In the second week of `December 2015` and `May 2015`, a total of `560` and `395` bugs are modified. There are many weeks in which `0` bugs are created this happened mostly from the year `2017` to `2019` and `2010` to `2013`.
+<font>
+
+# Conclusion
+
+<font size = 4>
+    In this project, I've visualized the bugRzilla database creating various visualization. The major graphs are the `time-series graph` which is the relationship of `bug_id`, `creation_ts`, `delta_ts`, and `last_modified` to show when the bug was created, last modified & last email to the cc list, last modified respectively. Some time-series graphs can help us to see in which week, month, the year most bugs are created.
+
+The `bar` graph which is the relationship between the `bug_count`, `bug_status`, `bug_severity`, `resolution` to check the status like closed, new, etc, how severe the bug is like whether the bug is critical, major, etc and the resolution of the bug-like if the bug is in a closed state, e.g. FIXED, DUPLICATE, etc.
+
+If your Bug report is effective, then its chances to get fixed are higher. It totally depends upon how effectively one reports it. Like some values are left and that should not be null to report the bugs effectively. For example, the `Priority`, `bug severity` even the `resolution` of the bugs should be specified. From the bugs data table, we can see that the column `bug_file_loc` is mostly empty like there should be proper reporting where the bug occurred. Event `Priority` is generally set from `P1` to `P5.` `P1` as “fix the bug with the highest priority” and `P5` as ” Fix when time permits” There are many rows where the priority is not mentioned which in turns takes more time by the developer to set the priority. At last in the description of the bug, these points should be there which can create the report much effective i.e.
```

Comment:
`priority` and `resolution` are set by R Core, not bug reporters. `bug_file_loc` is missing in `bugs_mod_df` but not in `bugs_df` - what values does this take? Not sure if this is something the user specifies - see screenshot below for what user inputs

![image](https://user-images.githubusercontent.com/3343008/129797162-73f96dd1-cc9e-40af-b92c-902ea18a61b0.png)